Scripts/Source/minai_Sex.psc

```diff
@@ -84,12 +93,14 @@ Function Maintenance(minai_MainQuestController _main)
   if !minai_UseOStim
     Main.Error("Could not find ostim toggle")
   EndIf
+
+  useOstim = bHasOstim && minai_UseOStim.GetValue() == 1.0
```

Comment:
Users that disable ostim won't have their change apply until the next time they save/load. Probably not a huge deal, we can fix that by adding an MCM option for this and having that update this variable.

Reply:
made `useOstim` function which will check those variables each time when called, instead of one in maintenance